Build failing via maven-assembly-plugin: group id is too big

I am getting a build failure on RHEL 7.9 and Ubuntu 18.04 that looks to be introduced with the upgrade of  maven-assembly-plugin (#414) regarding the group id being too big.

```
[INFO] Building tar: /tmp/heritrix3/dist/target/heritrix-3.4.0-SNAPSHOT-dist.tar.gz
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Heritrix 3 3.4.0-SNAPSHOT:
[INFO] 
[INFO] Heritrix 3 ......................................... SUCCESS [  0.001 s]
[INFO] Heritrix 3: 'commons' subproject (utility classes) . SUCCESS [01:05 min]
[INFO] Heritrix 3: 'modules' subproject (reusable components) SUCCESS [01:47 min]
[INFO] Heritrix 3: 'engine' subproject .................... SUCCESS [01:29 min]
[INFO] Heritrix 3 (distribution bundles) .................. FAILURE [ 22.762 s]
[INFO] Heritrix 3: 'contrib' subproject ................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:45 min
[INFO] Finished at: 2021-12-03T11:42:12-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.3.0:single (job.assembly.package) on project heritrix: Execution job.assembly.package of goal org.apache.maven.plugins:maven-assembly-plugin:3.3.0:single failed: group id '1000200513' is too big ( > 2097151 ). Use STAR or POSIX extensions to overcome this limit -> [Help 1]
```

It looks like this can be fixed by using `<tarLongFileMode>posix</tarLongFileMode>`.

--- a/contrib/pom.xml
+++ b/contrib/pom.xml
@@ -123,6 +123,7 @@
 							src/main/assembly/dist.xml
 						</descriptor>
 					</descriptors>
+					<tarLongFileMode>posix</tarLongFileMode>
 				</configuration>
 				<executions>
 					<execution>

--- a/contrib/src/main/assembly/dist.xml
+++ b/contrib/src/main/assembly/dist.xml
@@ -12,12 +12,17 @@
 	</dependencySets>
 	<fileSets>
 		<fileSet>
-			<directory>.</directory>
+			<directory>..</directory>
 			<outputDirectory>/</outputDirectory>
 			<includes>
-				<include>../dist/README.txt</include>
-				<include>../dist/LICENSE.txt</include>
-				<include>../dist/HOWTO-Launch-Heritrix.txt</include>
+				<include>README*</include>
+			</includes>
+		</fileSet>
+		<fileSet>
+			<directory>../dist</directory>
+			<outputDirectory>/</outputDirectory>
+			<includes>
+				<include>LICENSE.txt</include>
 			</includes>
 		</fileSet>
 		<fileSet>

